### PR TITLE
Refactor for consistency

### LINKED ISSUE #21: Refactor for consistency
Last package to refactor for consistency. Includes
- [x] no bumpversion
- [x] setup.py
- [x] `__init__.py`
- [x] docs/conf.py
- [x] no setup_requires
- [x] direct use of pytest
- [x] update license to actually be MIT (as said in setup.py)
- [x] update out-of-date parts of README

### PR DESCRIPTION
**Pull request by fsoubelet**
Restructures the common files according to conventions in pylhc packages. Fixes #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,57 +5,41 @@
...
-
-Reading and writing capabilities for [sdds-files](https://ops.aps.anl.gov/SDDSIntroTalk/slides.html)
-are provided by this package. On the python side, the data is stored in a class structure
-with attributes corresponding to the sdds-format itself 
-(see [sdds-format](https://ops.aps.anl.gov/manuals/SDDStoolkit/SDDStoolkitsu2.html)).
-
-
 ## Known Issues
 
-- Can't read ASCII files
```
Comment (lmalina) [inline README.md:45]: I was lazy to fix this for quite a while, nice

File: doc/conf.py [hunk 2]
```diff
@@ -52,24 +65,24 @@
...
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = u'sdds'
-copyright = u'2019, OMC-TEAM'
-author = u'OMC-TEAM'
+project = ABOUT_SDDS["__title__"]
+copyright = "2019-2020, pyLHC/OMC-TEAM"
```
Comment (lmalina) [inline doc/conf.py:75]: years also here...

File: setup.py [hunk 3]
```diff
@@ -1,70 +1,71 @@
...
-    with open(os.path.join(
-        os.path.dirname(__file__), MODULE_NAME, '__init__.py'
-    )) as init:
-        for line in init.readlines():
-            res = re.match(r'^__version__ = [\'"](.*)[\'"]$', line)
-            if res:
-                return res.group(1)
+# Extra dependencies
+EXTRA_DEPENDENCIES = {
+    "test": ["pytest>=5.2", "pytest-cov>=2.6", "hypothesis>=3.23.0", "attrs>=19.2.0"],
```
Comment (lmalina) [inline setup.py:37]: not sure we need hypothesis

**Review by lmalina — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fsoubelet) [reply README.md:45]: :)

File: setup.py (hunk 3 above)
Comment (fsoubelet) [reply setup.py:37]: I've been saying this for ages 😭

**Review by lmalina — APPROVED** [review]

**Pull request merged**